soh/soh/Enhancements/randomizer/settings.cpp

```diff
@@ -541,6 +546,201 @@ void Settings::CreateOptions() {
+    mOptionGroups[RSG_ITEM_POOL_HINTS_IMGUI_COLUMN] = OptionGroup::SubGroup("Item Pool & Hints", std::initializer_list<OptionGroup*>{
+        &mOptionGroups[RSG_ITEM_POOL_HINTS_IMGUI],
+        &mOptionGroups[RSG_EXTRA_HINTS_IMGUI],
+    }, false, WidgetContainerType::COLUMN);
+    mOptionGroups[RSG_ADDITIONAL_FEATURES_IMGUI] = OptionGroup::SubGroup("Additional Features", {
+        &mOptions[RSK_FULL_WALLETS],
+        &mOptions[RSK_BOMBCHUS_IN_LOGIC],
+        &mOptions[RSK_ENABLE_BOMBCHU_DROPS],
+        &mOptions[RSK_BLUE_FIRE_ARROWS],
+        &mOptions[RSK_SUNLIGHT_ARROWS]
+    }, false, WidgetContainerType::COLUMN);
+    mOptionGroups[RSG_GAMEPLAY_IMGUI_TABLE] = OptionGroup::SubGroup("Gameplay", {
+        &mOptionGroups[RSG_TIMESAVERS_IMGUI],
+        &mOptionGroups[RSG_ITEM_POOL_HINTS_IMGUI_COLUMN],
+        &mOptionGroups[RSG_ADDITIONAL_FEATURES_IMGUI]
+    }, false, WidgetContainerType::TABLE);
+    mOptionGroups[RSG_STARTING_EQUIPMENT_IMGUI] = OptionGroup::SubGroup("Starting Equipment", {
+        &mOptions[RSK_LINKS_POCKET],
+        &mOptions[RSK_STARTING_KOKIRI_SWORD],
+        &mOptions[RSK_STARTING_MASTER_SWORD],
+        &mOptions[RSK_STARTING_DEKU_SHIELD]
+    }, false, WidgetContainerType::COLUMN);
+    mOptionGroups[RSG_STARTING_ITEMS_IMGUI] = OptionGroup::SubGroup("Starting Items", {
+        &mOptions[RSK_STARTING_OCARINA],
+        &mOptions[RSK_STARTING_CONSUMABLES],
+        &mOptions[RSK_STARTING_SKULLTULA_TOKEN],
+    }, false, WidgetContainerType::COLUMN);
+    mOptionGroups[RSG_STARTING_NORMAL_SONGS_IMGUI] = OptionGroup::SubGroup("Normal Songs", {
+        &mOptions[RSK_STARTING_ZELDAS_LULLABY],
+        &mOptions[RSK_STARTING_EPONAS_SONG],
+        &mOptions[RSK_STARTING_SARIAS_SONG],
+        &mOptions[RSK_STARTING_SUNS_SONG],
+        &mOptions[RSK_STARTING_SONG_OF_TIME],
+        &mOptions[RSK_STARTING_SONG_OF_STORMS],
+    }, false, WidgetContainerType::SECTION);
+    mOptionGroups[RSG_STARTING_WARP_SONGS_IMGUI] = OptionGroup::SubGroup("Warp Songs", {
+        &mOptions[RSK_STARTING_MINUET_OF_FOREST],
+        &mOptions[RSK_STARTING_BOLERO_OF_FIRE],
+        &mOptions[RSK_STARTING_SERENADE_OF_WATER],
+        &mOptions[RSK_STARTING_REQUIEM_OF_SPIRIT],
+        &mOptions[RSK_STARTING_NOCTURNE_OF_SHADOW],
+        &mOptions[RSK_STARTING_PRELUDE_OF_LIGHT]
+    }, false, WidgetContainerType::SECTION);
+    mOptionGroups[RSG_STARTING_SONGS_IMGUI] = OptionGroup::SubGroup("Starting Songs", std::initializer_list<OptionGroup*>({
+        &mOptionGroups[RSG_STARTING_NORMAL_SONGS_IMGUI],
+        &mOptionGroups[RSG_STARTING_WARP_SONGS_IMGUI],
+    }), false, WidgetContainerType::COLUMN);
+    mOptionGroups[RSG_STARTING_INVENTORY_IMGUI_TABLE] = OptionGroup::SubGroup("Starting Inventory", {
+        &mOptionGroups[RSG_STARTING_EQUIPMENT_IMGUI],
+        &mOptionGroups[RSG_STARTING_ITEMS_IMGUI],
+        &mOptionGroups[RSG_STARTING_SONGS_IMGUI]
+    }, false, WidgetContainerType::TABLE);
```

Comment:
See [this comment](https://github.com/HarbourMasters/Shipwright/pull/3391/files#r1408104983) about the duplication of `OptionGroup` contents. Removing the duplication would require lots of spoiler parsing changes.